qemu/tests/cfg/expose_host_mtu.cfg

```diff
@@ -9,8 +15,7 @@
     mtu_value = 4000
     nic_extra_params = ",host_mtu=${mtu_value}"
     nm_stop_cmd = systemctl stop NetworkManager
-    check_guest_mtu_cmd = "ip link show dev %s"
-    set_mtu_cmd = "ip link set dev %s mtu ${mtu_value}"
+    check_linux_mtu_cmd = "ip link show dev %s"
+    set_mtu_cmd = "ip link set dev %s mtu %s"
```

Comment:
@xiywang This line is not used any more. I would suggest to remove it.

Reply:
@pezhang Updated.